package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-sfi-frontend",
-  "version": "1.17.1",
+  "version": "1.17.2",
```

Comment:
A bit more than a bug fix I reckon!

Reply:
Indeed. The intention was for it to be smaller! Updated to `1.18.0`